docs_user/modules/assembly_ospdo-scale-down-pre-database-adoption.adoc

```diff
@@ -0,0 +1,215 @@
+[id="ospdo_scale_down_pre_database_adoption_{context}"]
+
+= OSPdO scale down pre database adoption
+
+This section scales down and removes OSPdO resources in favor of the RHOSO ones.
+
+.Procedure
+
+. Scale-down RHOSO OpenStack Operator controller-manager to 0 replicas and delete its OpenStackControlPlane’s OpenStackClient pod temporarily, so that we can use the OSPdO controller-manager to clean up some of its resources, needed because OSPdO ultimately can’t act due to a pod name collision between its OpenStackClient and RHOSO’s OpenStackClient.
++
+----
+oc patch csv -n openstack-operators openstack-operator.v0.0.1 --type json   -p="[{"op": "replace", "path": "/spec/install/spec/deployments/0/spec/replicas", "value": "0"}]"
+oc delete openstackclients.client.openstack.org --all
+----
++
```

Comment:
```suggestion
. Scale down the {rhos_acro} OpenStack Operator `controller-manager` to 0 replicas and temporarily delete the `OpenStackControlPlane` `OpenStackClient` pod, so that you can use the OSPdO `controller-manager` to clean up some of its resources. The cleanup is needed to avoid a pod name collision between the OSPdO OpenStackClient and the {rhos_acro} OpenStackClient.
+
----
$ oc patch csv -n openstack-operators openstack-operator.v0.0.1 --type json   -p="[{"op": "replace", "path": "/spec/install/spec/deployments/0/spec/replicas", "value": "0"}]"
$ oc delete openstackclients.client.openstack.org --all
----

```

Reply:
ok